src/fsharp/FSharp.Compiler.Unittests/FSharp.Compiler.Unittests.fsproj

```diff
@@ -49,10 +50,6 @@
     <WarningLevel>3</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <!-- need full name and SpecificVersion = true in order to convince msbuild to allow this reference when targeting portable47 -->
-    <Reference Include="nunit.framework, Version=2.6.3.13283, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77" Condition="'$(TargetFramework)' != 'sl5' AND '$(TargetFramework)' != 'sl3-wp'">
-      <SpecificVersion>true</SpecificVersion>
-    </Reference>
     <Reference Include="NUnitFramework" Condition="'$(TargetFramework)' == 'sl5' OR '$(TargetFramework)' == 'sl3-wp'" />
```

Comment:
Could this reference also be removed?
It will be conditionally added in the new `FSharpSource.Nuget.targets`, right?